editions/tw5.com/tiddlers/widgets/KeyboardWidget.tid

```diff
@@ -30,6 +30,7 @@ The content of the `<$keyboard>` widget is rendered normally. The keyboard short
 |!Variables |!Description |
 |`event-key` |The <<.var event-key>> variable contains the character, if possible. eg: `1`. You can experiment with some settings at: [[Key Codes (Example)]] |
 |`event-code` |The <<.var event-code>> variable contains a character description. eg:  `Digit1` instead of `1`. Or `Space` instead of an empty string ` `, which is hard to see|
+|`event-key-descriptor` |The <<.var event-key-descriptor>> variable is available if the keyboard event captured was configured using a [[keyboard shortcut descriptor|Keyboard Shortcut Descriptor]] of the form `((my-shortcut))` which references a configuration tiddler. |
```

Comment:
Do we need a `<<.from-version>>` flag here?

Reply:
@Jermolene on the line above the table there is a `from-version` flag covering the entire table as all of these variables have been introduced for 5.2.0